projects/aca-content/src/lib/components/common/user-info/user-info.component.spec.ts

```diff
@@ -23,18 +23,18 @@
  */
 
 import { ComponentFixture, TestBed } from '@angular/core/testing';
-import { AuthenticationService } from '@alfresco/adf-core';
+import { AuthenticationService, IdentityUserService } from '@alfresco/adf-core';
 import { PeopleContentService } from '@alfresco/adf-content-services';
 import { UserInfoComponent } from './user-info.component';
 import { AppTestingModule } from '../../../testing/app-testing.module';
 import { of } from 'rxjs';
 
-describe('UserInfoComponent', () => {
+describe('parseDisplayName', () => {
```

Comment:
this should be the name of the file, so UserInfoComponent

Reply:
I've changed it